cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
Depends on the setup of your tool chain.  It shouldn't take you longer than a few minutes to submit a few fixes as a fix PR.  Otherwise please have a look how you can do this faster.  The point is: please don't mix fixes and new features.